examples/configs/evaluation/lm_harness_hf_config.yaml

```diff
@@ -4,7 +4,7 @@ model:
   torch_dtype: "bfloat16"
 
 # Settings specific to lm_harness.evaluate
-evaluator:
+evaluation:
```

Comment:
This was changed to be consistent with the prometheus config, which names it `evaluation` instead of `evaluator`